plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -623,10 +623,39 @@ items:
     - create
     - patch
     - update
+  - apiGroups:
+    - coordination.k8s.io
+    resources:
+    - leases
+    verbs:
+    - create
+  - apiGroups:
+    - coordination.k8s.io
+    resourceNames:
+    - kube-controller-manager
+    resources:
+    - leases
+    verbs:
+    - get
+    - update
```

Comment:
and patch?

Reply:
we don't use it.

Reply:
and actually, we probably don't want people patching this resource... we want them providing updates with resourceVersions so we know they saw the latest state

Reply:
+1